.github/actions/setup_environment/action.yml

```diff
@@ -97,38 +93,36 @@ runs:
     # - uses: syphar/restore-pip-download-cache@v1
     #   with:
     #     requirement_files: 'pyproject.toml'
-    #     # The pip download cache can be updated on a weekly basis as new packages
-    #     # don't appear that often
+    #  # The pip download cache can be updated on a weekly basis as new packages
+    #  # don't appear that often
     #     custom_cache_key_element: ${{ inputs.cache_version }}-${{ inputs.python-version }}-${{steps.date.outputs.week}}
-
     - name: Install Terraform (Windows)
       if: ${{ inputs.os == 'windows-latest' }}
       shell: bash
       run: choco install terraform -y
-
     - name: Install Terraform (Mac)
       if: ${{ inputs.os == 'macos-latest' }}
       shell: bash
       run: |
         brew tap hashicorp/tap
         brew install hashicorp/tap/terraform
-
     - name: Install ZenML and dependencies
-      if: steps.cache-virtualenv.outputs.cache-hit != 'true' && steps.custom-cache-pip.outputs.cache-hit != 'true'
+      if: steps.cache-virtualenv.outputs.cache-hit != 'true' && steps.custom-cache-pip.outputs.cache-hit
+        != 'true'
       shell: bash
       run: |
         scripts/install-zenml-dev.sh --integrations ${{ inputs.install_integrations }}
 
       # if using a cached virtualenv, just refresh the ZenML installation 
     - name: Refresh ZenML installation
-      if: steps.cache-virtualenv.outputs.cache-hit == 'true' || steps.custom-cache-pip.outputs.cache-hit == 'true'
+      if: steps.cache-virtualenv.outputs.cache-hit == 'true' || steps.custom-cache-pip.outputs.cache-hit
```

Comment:
does GitHub detect this back-to-line without an issue?

Reply:
@coderabbitai wdyt?